### DISCUSSION TITLE
error: linking with `cc` failed

### DISCUSSION THREAD

**Discussion by mateusfg7** [tauri-apps/tauri#8573] [Q&A]
I just start a new project using this guide: https://tauri.app/v1/guides/getting-started/setup/sveltekit#create-the-frontend.

Setup all steps from this guide but when I try to run `pnpm tauri dev` I got this error:

https://gist.github.com/mateusfg7/e69c4db96d7fd3ed32a092f63c1487da

**Package Manager**: PNPM 8.8.0
**Rust**: rustc 1.75.0 (82e1608df 2023-12-21)
**Svelte**: 5.0.0-next31
**Typescript**: 5.3.3
**SO**: Manjaro Linux x86_64
**Kernel**: 6.1.64-1-MANJARO
**DE**: GNOME 45.1

I got the same error when trying to start the app from `pnpm create tauri-app` generated app.

**Svelte**: 4.2.8

The error persist with a React app generated with `pnpm create tauri-app`.

Looks like the error is related with a missing lib.

**Comment by FabianLars** [comment]
Hmm, 3 questions to start with:
- You did install all the prereqs from here? https://tauri.app/v1/guides/getting-started/prerequisites#setting-up-linux
- Do you have homebrew installed? If so, read this: https://tauri.app/v1/guides/faq#build-conflict-with-homebrew-on-linux
- What type of terminal are you running those commands in? vscode's built-in for example? If so, try a "native" terminal instead.

**Reply by mateusfg7** [reply]
1. I have installed all the prereqs from Arch example, replacing `pacman` with `yay`.
2. I don't have homebrew installed.
3. I'm using gnome-terminal.

**Comment by FabianLars** [comment]
https://github.com/tauri-apps/tauri/issues/8594
